website/docs/r/notifylist.html.markdown

```diff
@@ -28,6 +28,13 @@ resource "ns1_notifylist" "nl" {
       email = "test@test.com"
     }
   }
+  
+ notifications {
```

Comment:
Looks like indentation is off?  This doesn't appear to line up with the block above it.